distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Normalization.enso

```diff
@@ -0,0 +1,57 @@
+import project.Any.Any
+
+import project.Data.Vector.Vector
+import project.Metadata.Display
+import project.Metadata.Widget
+from project.Metadata.Choice import Option
+from project.Metadata.Widget import Single_Choice
+
+polyglot java import com.ibm.icu.text.Normalizer2.Mode
+
+## Enso representations of `com.ibm.icu.text.Normalizer2.Mode`. Does not
+   include all values of `Mode`.
+type Normalization
+    # Unicode standard NFC
+    NFC
```

Comment:
Please make sure these comments are visible in the Component Browser when you try to select the particular constructor.

I think that you need to use `##` for them to be visible - as `##` is treated not only as a multiline comment block but also a doc comment, whereas I think that `#` may be discarded - unless something has changed since I remember that.

Reply:
Done.